__init__.py

```diff
@@ -376,21 +386,28 @@ def get_harmonic_thermo(self, hnu):
         ZPE_hmodes = []  # Zero point energy for mode
         Z_hmodes = []  # partition function for mode
         e_excitation_hmodes = []  # principle energy of mode
+        entropic_energy_hmodes = [] #entropy contribution per mode
+        E_kT = 0.
 
         for e in hnu:
             if e.imag == 0 and e.real >= 0.010:
                 Z_mode = 1./(1.-np.exp(-e.real/(self.kT)))
                 ZPE = 0.5 * e.real
                 e_min_exitation = e.real
+                E_kT = e_min_exitation/self.kT
```

Comment:
Same as earlier. x is better than E_kT as I think calling it E_kT makes it more confusing than x.